### PR TITLE
Ad 53 registration/files and basic logic

### PR DESCRIPTION
**Pull request by BartoszBilejczyk**

### CODE REVIEW

File: src/app/auth/auth.component.html [hunk 1]
```diff
@@ -1 +1,47 @@
...
+                    <span>AI Ready</span>
+                </div>
+            </div>
+
+
+            <img class="adsh-auth__poster-logo"
+                 src="assets/images/logo--white.png"
+                 srcset="assets/images/logo--white@2x.png 2x,
+                         assets/images/logo--white@3x.png 3x"
+                 alt="Adshares Logo">
```
Comment (KonradFak) [inline src/app/auth/auth.component.html]: I thought we agreed to keep all attributes in new lines:
```
 <img 
     class="adsh-auth__poster-logo"
     src="assets/images/logo--white.png"
     srcset="assets/images/logo--white@2x.png 2x,
                   assets/images/logo--white@3x.png 3x"
     alt="Adshares Logo">
```
Except for case when there will be only one html element attribute, just as you did above this `img` tag

File: src/app/auth/auth.service.ts [hunk 2]
```diff
@@ -11,4 +11,9 @@ export class AuthService {
   loginUser(email: string, password: string) {
     return this.http.post(`${environment.apiUrl}/user`, { email, password });
   }
+
+  registerUser(email: string, password: string) {
+    // http TODO
```
Comment (KonradFak) [inline src/app/auth/auth.service.ts]: I think u can remove this todo comment

File: src/app/auth/confirmation/confirmation.component.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<div class="col
+            justify-between
+            adsh-confirmation-wrapper">
+  <div>
+    <a routerLink="/auth/login"
+       class="row
+              align-center">
```
Comment (KonradFak) [inline src/app/auth/confirmation/confirmation.component.html]: Same with new line attributes

File: src/app/auth/confirmation/confirmation.component.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-confirmation',
+  templateUrl: './confirmation.component.html',
+  styleUrls: ['./confirmation.component.scss'],
+})
+export class ConfirmationComponent implements OnInit {
+  email: string = 'mateuszpiatek@10clouds.com'
```
Comment (KonradFak) [inline src/app/auth/confirmation/confirmation.component.ts]: This is also unnecessary I think ;)

File: src/app/auth/login/login.component.html [hunk 5]
```diff
@@ -1,48 +1,83 @@
...
+                        justify-between
+                        adsh-login__remember">
+                <div class="row
+                            align-center">
+                    <input
+                        #rememberUser
+                        id="remember-user"
+                        type="checkbox">
+                    <label for="remember-user"
+                           class="checkbox-label"></label>
```
Comment (KonradFak) [inline src/app/auth/login/login.component.html]: New line attributes

File: src/app/auth/login/login.component.html [hunk 6]
```diff
@@ -1,48 +1,83 @@
...
+                    <span class="adsh-copy
+                                 adsh-copy--large
+                                 adsh-copy--x-dark">Remember me</span>
+                </div>
 
-        <a routerLink="/auth/remind-password">Forgot Password?</a>
+                <a class="adsh-copy
+                          adsh-copy--large"
+                   routerLink="/auth/remind-password">Forgot Password?</a>
```
Comment (KonradFak) [inline src/app/auth/login/login.component.html]: Also here

File: src/app/auth/login/login.component.html [hunk 7]
```diff
@@ -1,48 +1,83 @@
...
+            </div>
 
-        <button type="submit">Sign In</button>
-    </form>
+            <button type="submit"
+                    class="adsh-btn
+                           adsh-btn--wide
+                           adsh-bg-blue
+                           adsh-form-submit-btn">Sign In</button>
```
Comment (KonradFak) [inline src/app/auth/login/login.component.html]: And here

File: src/app/auth/register/register.component.html [hunk 8]
```diff
@@ -1,3 +1,84 @@
...
+                  class="adsh-form-input"
+                  type="email"
+                  name="email"
+                  ngModel
+                  [email]="true"
+                  #email="ngModel"
+                  required>
+              <span *ngIf="email.valid" class="input-valid"></span>
+              <span *ngIf="email.hasError('email') && email.touched">Invalid email!</span>
+              <span *ngIf="email.hasError('required') && formSubmitted">Email required!</span>
```
Comment (KonradFak) [inline src/app/auth/register/register.component.html]: `formSubmitted` should be removed from component, pls use here `registrationForm.submitted`

File: src/app/auth/register/register.component.html [hunk 9]
```diff
@@ -1,3 +1,84 @@
...
+                  type="password"
+                  ngModel
+                  #password="ngModel"
+                  minlength="8"
+                  required>
+              <span *ngIf="password.valid" class="input-valid"></span>
+              <span *ngIf="password.hasError('minlength') && password.touched">
+                  Minimum 8 signs required!
+              </span>
+              <span *ngIf="password.hasError('required') && formSubmited">Password required</span>
```
Comment (KonradFak) [inline src/app/auth/register/register.component.html]: Remove `formSubmited `

File: src/app/auth/register/register.component.html [hunk 10]
```diff
@@ -1,3 +1,84 @@
...
+          <button
+              type="submit"
+              class="adsh-btn
+                     adsh-btn--wide
+                     adsh-bg-blue
+                     adsh-form-submit-btn">Sign Up</button>
+      </form>
+      <p class="adsh-copy
+                adsh-copy--small
+                adsh__registration-consent">By clicking "Sign Up" I agree to Adshares <a class="adsh-blue" href="#">Terms of Service</a> and <a class="adsh-blue" href="#">Privacy Policy.</a></p>
```
Comment (KonradFak) [inline src/app/auth/register/register.component.html]: This is quite long line, maybe move content of paragraph to new line with proper indent

File: src/app/auth/register/register.component.ts [hunk 11]
```diff
@@ -1,12 +1,36 @@
...
+import { AuthService } from '../auth.service'
 
 @Component({
   selector: 'app-register',
   templateUrl: './register.component.html',
   styleUrls: ['./register.component.scss'],
 })
 export class RegisterComponent {
+  @ViewChild('registrationForm') registrationForm: NgForm;
+  formSubmitted = false;
```
Comment (KonradFak) [inline src/app/auth/register/register.component.ts]: `formSubmitted` is unnecessary, as I wrote in one of comment above

File: src/app/auth/auth.component.html [hunk 12]
```diff
@@ -1 +1,48 @@
...
+                    <span>Censorship Resistant</span>
+                    <div class="adsh-circle
+                                adsh-circle--x-small
+                                adsh-bg-blue"></div>
+                    <span>AI Ready</span>
+                </div>
+            </div>
+
+
+            <img
```
Comment (aqum) [inline src/app/auth/auth.component.html]: Just a suggestion. I like how react styleguide describes spacing for props/attributes - it's more clean and predictable. What do you think? https://github.com/airbnb/javascript/tree/master/react#alignment

Comment (aqum) [reply src/app/auth/auth.component.html]: And I see that in HTML you have 4 spaces and in JS 2. Which one is correct?

**Pull request merged**